docs/using-radicle/fetching-and-merging.md

```diff
@@ -0,0 +1,142 @@
+---
+id: fetching-and-merging
+title: Fetching and merging contributions
+---
+
+To be able to collaborate on a Radicle project, you need to know how to fetch
+and merge contributions from other people. To fetch and merge changes from a
+contributor, you first need to add them as a remote to your project. If you
+haven't done so, then first follow the steps in [Tracking & viewing
+contributions][tv] section.
+
+If you can see your a tracked peer's changes after adding them as a remote, this
+means they have been successfully replicated to your machine. If you have added
+a peer as a remote and can't see their changes, head over to our
+[Troubleshooting][tr] section.
+
+## Fetching changes 
+
+If you're interested in merging these changes into one of your branches, you'll
+need to fetch these commits into your working copy. To make sure you can fetch or push changes, you need to add the
+[`git-rad-remote` helper][gs] to your `$PATH`.
+
+Navigate to your woking copy in a terminal. Here you can inspect the remotes of
+your project by running the `git remote` command. For example, a project that
+tracks multiple people might look something like this.
+
+```
+$ git remote show
+fintohaps@hybaapjdrhj8mg4mathpwjaeuktfzxd8uqfpjknp65fyncehpxxzze
+merle@hyb1exienq5txww75rgdftuf6j78qehtigma6hn19zt8gz5pu5k73g
+rad
+xla@hydxmk8cx8mcbijp6kgb8htm6jdhpstiow3kbezxp438adybaaqzgy
```

Comment:
```suggestion
```

Loosk messy otherwise and the message still comes across. Thanks for including me. :>